qcodes/instrument/parameter.py

```diff
@@ -2042,12 +2094,13 @@ def __init__(self,
         if not xor(is_divider, is_amplifier):
             raise ValueError('Provide only division OR gain')
 
-        if is_divider:
+        if division is not None:
             self.role = ScaledParameter.Role.DIVISION
-            self._multiplier = division
-        elif is_amplifier:
+            # mypy issue 3004
```

Comment:
Flesh out the comment to reflect why mypy issue 3004 means that this cannot be done probably